app-services/secrets/res/configuration.toml

```diff
@@ -52,17 +52,43 @@ Host = "localhost"
 Port = 27017
 Timeout = "30s"
 
+# Shared Secret Store
 [SecretStore]
-  Host = 'localhost'
+  # Host = 'localhost'
+  Host = 'edgex-vault'
```

Comment:
This goes against the new config strategy, which is default config is for running native and we use env overrides in the compose files for use in docker. So this particular change is already  handled by the global overrides in our compose files.